moveit_ros/moveit_servo/tests/test_independent_utils.cpp

```diff
@@ -0,0 +1,78 @@
+
+
+#include <gtest/gtest.h>
+#include <tf2_eigen/tf2_eigen.hpp>
+
+#include <moveit_servo/datatypes.hpp>
+#include <moveit_servo/utils.hpp>
+
+TEST(ServoUtilsUnitTests, validVector)
+{
+  Eigen::VectorXd valid_vector(7);
+  valid_vector << 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0;
```

Comment:
Not possible if using `VectorXd`,  but since we already know the number of joints for panda robot, I changed this to 
`vector<double, 7>` which will let us initialize directly.